### DISCUSSION TITLE
Is there a problem with the `Pair Extraordinaire` achivement?

### DISCUSSION THREAD

**Discussion by devkennyy** [community/community#20638] [Other Feature Feedback, Questions, & Ideas]
According to the criteria, all you need is to be a co-author on a commit. 
![image](https://user-images.githubusercontent.com/105208736/179322751-e9112f41-a0c5-43ec-b81f-44eeafe73789.png)
The first author doesn't have the achievement and I'm missing the second tier. 
I assume you need at least 4 people in a commit because that's how I received the first tier.

**Comment by solyasa** [comment]
https://github.com/drknzz/GitHub-Achievements

**Comment by airtower-luna** [comment]
You need to be co-author in a commit, the commit must become part of a PR, and then that PR must get merged. :wink:
